samples/compute/v2/create_server.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+require __DIR__ . '/../../../vendor/autoload.php';
+
+use OpenStack\OpenStack;
+
+$openstack = new OpenStack;
+
+$compute = $openstack->computeV2([
+    'region' => 'RegionOne',
+    'debug'  => true,
+]);
+
+$s = $compute->createServer('php_test', 'e37365c2-5c45-4b73-b4ae-828436d5c569', '1');
```

Comment:
I was 100% for assoc arrays before, but actually they're not great for IDE users - positional args can be a lot more explicit. Then again, we can mitigate this issue by documenting the parameters well.

Do you want to use assoc arrays for _all_ methods? It seems a bit verbose to ask users to do:

```php
$s = $compute->getServer(['id' => 'e37365c2-5c45-4b73-b4ae-828436d5c569']);
```

Maybe we could have a general rule of thumb: positional arguments can be used for <= 3 required parameters. For >3 required parameters, and for optional parameters, use an associative array. What do you think?